modules/default.nix

```diff
@@ -82,27 +102,75 @@ in
         description = "Whether to mount the ESP partition on eMMC under /opt/nvidia/esp on Xavier AGX platforms. Needed for capsule updates";
         internal = true;
       };
+
+      hostPkgs = mkOption {
```

Comment:
Additionally, I think this would need to have a different name to avoid confusion with `hostPlatform`, which would normally by `aarch64-linux`, not `x86_64-linux`.  Maybe `flasherPkgs`?

Reply:
Changed to `flasherPkgs`. The motivation for the `hostPkgs` name was to reuse similar logic as nixpkgs [here](https://github.com/nixos/nixpkgs/blob/900d8043bbf1c2b3eabef273b599aef0e4ffb2a8/nixos/lib/testing/driver.nix#L112), where the package set represented by that option is configured for the host running qemu, which is needed for running qemu on macos, but the guest being a linux machine. Obviously the semantics here are different, so it makes sense to have a different name.